gradle/plugins/common/src/main/kotlin/junitbuild.osgi-conventions.gradle.kts

```diff
@@ -38,6 +39,7 @@ tasks.withType<Jar>().named {
 				Import-Package: \
 					${importAPIGuardian},\
 					${importJSpecify},\
+					${importJdkJfr},\
```

Comment:
Please move these changes to the build script of junit-platform-launcher as they're specific to that module/bundle (see junit-platform-commons for an example).

Reply:
No worries! I pushed a few commits. I agree that modifying an existing definition is not as DRY as one would wish.